<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketMain</class>
 <widget class="QWidget" name="RocketMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="mainLayout" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="mapLayout" stretch="1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="mapFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border-radius: 25px;
	border: 5px solid rgb(212,235,242);
	background-color: white;
} 


</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QQuickWidget" name="map">
          <property name="geometry">
           <rect>
            <x>14</x>
            <y>14</y>
            <width>557</width>
            <height>318</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="resizeMode">
           <enum>QQuickWidget::ResizeMode::SizeViewToRootObject</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="altitudeFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border-radius: 25px;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 




</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QStackedWidget" name="altitudeStack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QCustomPlot" name="altitudeGraph"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="dataLayout" stretch="1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="LCDLayout" stretch="2,2,1,3">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QFrame" name="LCD1Frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background: transparent;
border: 0px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QGridLayout" name="LCD1Layout">
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="2" column="2">
               <widget class="QLCDNumber" name="altitudLCD">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLCDNumber
{
	color: white;
}

QFrame
{
	border: 0px, white;
	border-radius: 25px;
}</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="altitudLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI Black</family>
                  <pointsize>16</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: black;
</string>
                </property>
                <property name="text">
                 <string>Altitude [m]</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLCDNumber" name="timeLCD">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLCDNumber
{
	color: white;
}

QFrame
{
	border: 0px, white;
	border-radius: 25px;
}</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="timeLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI Black</family>
                  <pointsize>16</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Time [s]</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="LCD2Frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background: transparent;
border: 0px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="LCD2Layout">
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
               <widget class="QLabel" name="statusLCD">
                <property name="font">
                 <font>
                  <family>Segoe UI Semibold</family>
                  <pointsize>60</pointsize>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: white;</string>
                </property>
                <property name="text">
                 <string>Standby</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="statusLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI Black</family>
                  <pointsize>16</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Status</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="maxAltitudLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI Black</family>
                  <pointsize>16</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: black;
</string>
                </property>
                <property name="text">
                 <string>Apogeee [m]</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLCDNumber" name="maxAltitudLCD">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLCDNumber
{
	color: white;
}

QFrame
{
	border: 0px, white;
	border-radius: 25px;
}</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="orderFrame">
           <property name="styleSheet">
            <string notr="true">QFrame
{
	border: 0px;
	background-color: white;
} 




</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QListView" name="orderPrint">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
                <italic>false</italic>
                <bold>true</bold>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame
{
	border: 0px;
	background-color: white;
} 




</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="serialFrame">
           <property name="styleSheet">
            <string notr="true">#serialFrame
{
	border: 5px solid rgb(212,235,242);
	background-color: white;
	border-radius: 25px;
} 


</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QListView" name="serialPrint">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
                <italic>false</italic>
                <bold>true</bold>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame
{
	border: 0px;
	background-color: white;
} 




</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="artifactsLayout" stretch="5,0">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QGridLayout" name="gaugesLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="spacing">
            <number>20</number>
           </property>
           <item row="1" column="0">
            <widget class="QQuickWidget" name="speed">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="resizeMode">
              <enum>QQuickWidget::ResizeMode::SizeViewToRootObject</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="accLabel">
             <property name="font">
              <font>
               <family>Segoe UI Black</family>
               <pointsize>16</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;
</string>
             </property>
             <property name="text">
              <string>Acceleration [G]</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="gyroscopeLabel">
             <property name="font">
              <font>
               <family>Segoe UI Black</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;
</string>
             </property>
             <property name="text">
              <string>Gyroscope</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QQuickWidget" name="acc">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="resizeMode">
              <enum>QQuickWidget::ResizeMode::SizeViewToRootObject</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="speedLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI Black</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;
</string>
             </property>
             <property name="text">
              <string>Speed [m/s]</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QQuickWidget" name="gyroscope">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="resizeMode">
              <enum>QQuickWidget::ResizeMode::SizeViewToRootObject</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="buttonLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="1">
            <widget class="QPushButton" name="phase_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
             </property>
             <property name="text">
              <string>Phase</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="tare_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
             </property>
             <property name="text">
              <string>Tare</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="ping_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
             </property>
             <property name="text">
              <string>Ping</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="alarm_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
             </property>
             <property name="text">
              <string>Alarm</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>teston()</slot>
  <slot>ledon()</slot>
  <slot>ledoff()</slot>
  <slot>ignitionon()</slot>
  <slot>ignitionoff()</slot>
  <slot>paracason()</slot>
  <slot>paracasoff()</slot>
  <slot>alarmaon()</slot>
  <slot>alarmaoff()</slot>
  <slot>testoff()</slot>
  <slot>exporttest()</slot>
  <slot>exportgraph()</slot>
 </slots>
</ui>
